pset-iterators.js

```diff
@@ -93,6 +105,10 @@ console.log('Problem 3:')
 
 // Add your code below this line
 
+const presidentialParties = presidents.map(p => p.party)
+
+console.log(presidentialParties)
+
```

Comment:
@rlou88 Great job here using this one-liner map method to return only the president parties.

Take a look at this approach [here](https://github.com/jsr20190826/hw-03-iterators-solution/blob/96bab20deba3ad116374a5827291a951faf5d555/pset-iterators.js#L94)